main.go

```diff
@@ -59,7 +60,17 @@ import (
 	"github.com/percona/qan-api2/utils/logger"
 )
 
-const shutdownTimeout = 3 * time.Second
+const (
+	shutdownTimeout = 3 * time.Second
+	defaultDsnF     = "clickhouse://{{ .Hostname }}:{{ .Port }}?database={{ .DatabaseName }}&block_size=10000&pool_size=2"
```

Comment:
it can be done via fmt.sprintf or even url

Reply:
Yep, it's a good point